.github/workflows/pr.yaml

```diff
@@ -11,7 +11,7 @@ jobs:
       - name: check out pr branch
         uses: actions/checkout@v2
       - name: Run tests in tester
-        uses: ./.github/actions/test
+        run: docker build -t uat ./; docker run -v /var/run/docker.sock:/var/run/docker.sock uat sbt clean test
```

Comment:
Would you add a `run-test` script under `bin/` and update `README` how to run the tests locally?

Reply:
Sure thing!